api/proto/info/info.proto

```diff
@@ -278,6 +298,28 @@ message ZCellularModuleInfo {
     string firmware_version = 3;
 }
 
+// Device Run State
+enum ZDeviceState {
+    // Just to handle Un-Initialized runState variable Internally
+    ZDEVICE_STATE_UNSPECIFIED = 0;
+    // Device is Online. This only means all components of EVE are up.
+    // This does not indicate the Status of Apps or other configured objects.
+    // During booting up after reboot, Device might be reported as ONLINE
+    // while apps are still being processed.
+    ZDEVICE_STATE_ONLINE = 1;
+    // Device Reboot in Progress. Device is in the process of going down.
+    // Once the device comes back up and Eve is started, device reports the
+    // the state as "BOOTING"
+    ZDEVICE_STATE_REBOOTING = 2;
+    ZDEVICE_STATE_MAINTENANCE_MODE = 3;
+    // Device BaseOs Upgrade in Progress
```

Comment:
One remaining case where upgrade should be changed to update.

Reply:
Done.